services/bettersocial/fcmToken/sendReplyCommentNotification.js

```diff
@@ -25,11 +25,12 @@ const sendReplyCommentNotification = async (
     data: {
       feed_id: activity_id,
       type: 'feed'
-    }
+    },
+    token: userTargetToken?.token
   };
   if (userTargetToken) {
     messaging()
-      .sendToDevice(userTargetToken?.token, payload)
+      .send(payload)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of the `send` method does not handle potential errors that might occur during the sending process. It's important to add error handling here to ensure any issues are caught and handled appropriately.
```diff
     data: {
       feed_id: activity_id,
       type: 'feed'
     },
     token: userTargetToken?.token
    };
    if (userTargetToken) {
      messaging()
       .send(payload)
-      .then(() => {});
+      .then(() => {})
+      .catch((error) => {
+        console.error(`Failed to send notification: ${error.message}`);
+      });
   }
 };
```
This change will log an error message if there is a failure when sending the notification, which can be useful for debugging purposes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->